autoload/fzf/vim/complete.vim

```diff
@@ -47,8 +47,9 @@ else
 endif
 
 function! fzf#vim#complete#word(...)
+  let sources = empty(&dictionary) ? '/usr/share/dict/words' : substitute(&dictionary, ',', ' ', 'g')
   return fzf#vim#complete(s:extend({
-    \ 'source': 'cat /usr/share/dict/words'},
+    \ 'source': 'cat ' . sources},
```

Comment:
Since we don't know what's inside the user's `&dictionary`, we have to `shellescape` each entry.

```vim
let sources = empty(&dictionary) ? ['/usr/share/dict/words'] : split(&dictionary, ',')

... 'cat ' . join(map(sources, 'shellescape(v:val)'))
```

Reply:
it sounds good to me, let's go with the changes.